docs/game_sdk/Overlay.md

```diff
@@ -103,6 +103,29 @@ overlayManager.OpenActivityInvite(Discord.ActivityActionType.Join, (result) =>
 });
 ```
 
+## OpenGuildInvite
+
+Opens the overlay modal for joining a Discord guild, given its invite code. An invite code for a server may look something like `fortnite` for a verified server—the full invite being `discord.gg/fortnite`—or something like `rjEeUJq` for a non-verified server, the full invite being `discord.gg/rjEeUJq`.
+
+Returns a `Discord.Result` via callback.
+
+###### Parameters
+
+| name | type   | description                |
+| ---- | ------ | -------------------------- |
+| code | string | an invite code for a guild |
+
+###### Example
+
+````cs
+overlayManager.OpenGuildInvite("rjEeUJq", (result) =>
+{
+  if (result == Discord.Result.Ok)
+  {
+    Console.WriteLine("User is joining your guild!");
```

Comment:
We don't tell them if the user actually joined. We jsut tell them if the invite was valid, this also fires when they either close the modal or join.

Reply:
My bad—words weren't intended to assume they're auto-joining. Will change.